resources/faq.md

```diff
@@ -112,46 +74,13 @@ Meilisearch has the following types of errors:
 
 | Type            | Description                                                                                       |
 |-----------------|---------------------------------------------------------------------------------------------------|
-| invalid_request | This is due to an error in the user input. It is accompanied by the HTTP code `4xx`. |
+| invalid_request | This is due to an error in the user input. It is accompanied by the HTTP I am trying to add my documents `4xx`. |
```

Comment:
Is this correct?

Reply:
I don't remember making this change. This FAQ was removed from this page as part of #1809. I'll revert this